cleanlab/datalab/internal/adapter/imagelab.py

```diff
@@ -95,6 +98,14 @@ def _update_issues_imagelab(self, imagelab: "Imagelab", overlapping_issues: List
         new_columnns = list(set(imagelab.issues.columns).difference(self.issues.columns))
         self.issues = self.issues.join(imagelab.issues[new_columnns], how="outer")
 
+    def filter_based_on_max_prevalence(self, issue_summary: pd.DataFrame, max_num: int):
+        removed_issues = issue_summary[issue_summary["num_images"] > max_num]["issue_type"].tolist()
+        if len(removed_issues) > 0:
+            print(
+                f"Removing {', '.join(removed_issues)} from potential issues in the dataset as it exceeds max_prevalence={IMAGELAB_ISSUES_MAX_PREVALENCE}"
```

Comment:
why is this line never hit in unit tests?  seems like unit tests are lacking if so

Reply:
Added unit test